### DISCUSSION TITLE
Should we create a team?

### DISCUSSION THREAD

**Discussion by jemunm378** [scratchdit/scratchdit#16] [General]
Not really important, but maybe we should create a team so we can work on privately subjects.

**Comment by jemunm378** [comment]
@Redguy12 Not sure why you limited conversation to collaborators, but ok.

**Reply by cobaltt7** [reply]
i didn't...

**Reply by cobaltt7** [reply]
i just converted it from an issue to a discussion

**Reply by jemunm378** [reply]
Oh.

**Comment by cobaltt7** [comment]
done, @scratchdit
